core/src/main/java/org/teavm/model/util/ModelUtils.java

```diff
@@ -30,9 +30,23 @@
 import org.teavm.model.MethodReader;
 
 public final class ModelUtils {
+
+    private static String scalaMarker = ".scala";
+    private static String scalaInternalMarker = "$";
+
     private ModelUtils() {
     }
 
+    public static Boolean isScalaClass(ClassReader cls) {
```

Comment:
Is there any reason to return `java.lang.Boolean` instead of primitive `boolean`?

Reply:
No, there is not any reason) Will change it

Reply:
I've reverted it.